packages/generator/src/renderer.ts

```diff
@@ -52,18 +52,23 @@ interface IBuildConfig {
   shopperAuthApi: string;
   exchangeDeploymentRegex: RegExp;
 }
+
 /**
- * Information used to generate APICLIENTS.md.
+ * Api Document with metadata
+ * TODO: This should be moved into raml-toolkit
  */
-export interface IApiClientsInfo {
-  model: model.domain.WebApi;
-  family: string;
-  config: RestApi;
-}
+export type DocumentWithMetadataT = {
+  document: model.document.Document;
+  metadata: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
```

Comment:
consider using unknown - type-safe counterpart of any

Reply:
Unkown didn't work due to something else down stream.  And really just trying to remove logic for the migration